.gitignore

```diff
@@ -13,3 +13,5 @@
 
 # Maven
 target/
+
+node_modules/
```

Comment:
Please add a seperate .gitignore inside the new directory and ignore the node_modules in that file.